packages/core/src/amazonqGumby/chat/controller/controller.ts

```diff
@@ -220,7 +220,6 @@ export class GumbyController {
                 break
             case ButtonActions.STOP_TRANSFORMATION_JOB:
                 await stopTransformByQ(transformByQState.getJobId(), CancelActionPositions.Chat)
-                this.messenger.sendJobFinishedMessage(message.tabId, true)
```

Comment:
Was causing an extra message to show when you stop a job. "transformFinished" gets fired at end of job which already handles this.